tsdb/engine/tsm1/string_test.go

```diff
@@ -63,7 +63,7 @@ func Test_StringEncoder_Multi_Compressed(t *testing.T) {
 		t.Fatalf("unexpected encoding: got %v, exp %v", b[0], stringCompressedSnappy)
 	}
 
-	if exp := 47; len(b) != exp {
+	if exp := 51; len(b) != exp {
```

Comment:
I'd rather keep the length check and have the test fail so someone investigates why as opposed to possibly letting a data corruption bug sneak in quietly.